components/openindiana/ddu/ddu-library.p5m

```diff
@@ -22,11 +22,11 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 <transform file path=usr/ddu/scripts/.+ -> default mode 0555>
-<transform file path=usr/lib/python3.5/vendor-packages/DDU/.*\.py$ \
-	-> default pkg.tmp.autopyc false>
 
 depend type=require fmri=system/library/storage/libmpapi
 
+<include autopyc>
```

Comment:
Why is this needed?  `autopyc` should be added automatically - see `ips.mk`.

Reply:
Oh, I seem to have lost my change from inside the vagrant VM...

Reply:
Done